lms/views/lti/deep_linking.py

```diff
@@ -137,6 +132,34 @@ def file_picker_to_form_fields_v13(self):
             )
         }
 
+    @view_config(route_name="lti.deep_linking.form_fields.v11")
+    def file_picker_to_form_fields_v11(self):
```

Comment:
The LTI 1.1 path doesn't use the application instance we look-up in the init, so that query probably isn't getting us anything. This means it's should probably move into the 1.3 method.